pcsx2/DEV9/ATA/ATA_Transfer.cpp

```diff
@@ -210,7 +210,8 @@ void ATA::IO_SparseCacheLoad()
 	std::streampos orgPos = hddImage.tellp();
 	//Load into cache
 	hddImage.seekg(HddSparseStart, std::ios::beg);
-	hddImage.read((char*)hddSparseBlock.get(), readSize);
+	//Suppress Check buffer boundaries warning
+	hddImage.read((char*)hddSparseBlock.get(), readSize); /* Flawfinder: ignore */
```

Comment:
I suspect this is a false positive, but I have added additional bounds checking code into the function.

Not sure what further is needed, so I've chosen to add an ignore comment, I can remove the comment if that is preferred.